bigbluebutton-html5/imports/api/voice-users/server/methods/muteToggle.js

```diff
@@ -8,10 +8,6 @@ export default function muteToggle(credentials, userId) {
   const REDIS_CONFIG = Meteor.settings.private.redis;
   const CHANNEL = REDIS_CONFIG.channels.toAkkaApps;
   const EVENT_NAME = 'MuteUserCmdMsg';
-  const APP_CONFIG = Meteor.settings.public.app;
-  const ALLOW_MODERATOR_TO_UNMUTE_AUDIO = APP_CONFIG.allowModeratorToUnmuteAudio;
```

Comment:
Please remove the property `allowModeratorToUnmuteAudio` from settings.yml
https://github.com/bigbluebutton/bigbluebutton/blob/e6e40d8c84490a17df15fddb5c75caa3ca651a78/bigbluebutton-html5/private/config/settings.yml#L78